### PR TITLE
[PLAYER-2158] Added geoblocking assets into OoyalaSkinSampleApp

### PR DESCRIPTION
**Pull request by IvanSakharovskii**

### CODE REVIEW

File: OoyalaSkinSampleApp/OoyalaSkinSampleApp/Lists/GeoblockingTableViewController.h [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//
+//  GeoblockingTableViewController.h
+//  OoyalaSkinSampleApp
+//
+//  Created by Ivan Sakharovskii on 1/30/18.
```
Comment (akshaydch11) [inline OoyalaSkinSampleApp/OoyalaSkinSampleApp/Lists/GeoblockingTableViewController.h]: remove created by

File: OoyalaSkinSampleApp/OoyalaSkinSampleApp/Lists/GeoblockingTableViewController.m [hunk 2]
```diff
@@ -0,0 +1,66 @@
+//
+//  GeoblockingTableViewController.m
+//  OoyalaSkinSampleApp
+//
+//  Created by Ivan Sakharovskii on 1/30/18.
```
Comment (akshaydch11) [inline OoyalaSkinSampleApp/OoyalaSkinSampleApp/Lists/GeoblockingTableViewController.m]: remove created by

File: OoyalaSkinSampleApp/OoyalaSkinSampleApp/Players/GeoblockingSkinViewController.m [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  
+  // Create Ooyala ViewController
+  OOOptions *options = [OOOptions new];
+  OOOoyalaPlayer *ooyalaPlayer = [[OOOoyalaPlayer alloc] initWithPcode:self.pcode
+                                                                domain:[[OOPlayerDomain alloc] initWithString:self.playerDomain] embedTokenGenerator:self
+                                                               options:options];
+  OODiscoveryOptions *discoveryOptions = [[OODiscoveryOptions alloc] initWithType:OODiscoveryTypePopular limit:10 timeout:60];
+  NSURL *jsCodeLocation = [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
+  //  NSURL *jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle?platform=ios"];
+  ooyalaPlayer.actionAtEnd = OOOoyalaPlayerActionAtEndPause; //This is reccomended to make sure the endscreen shows up as expected
```
Comment (akshaydch11) [inline OoyalaSkinSampleApp/OoyalaSkinSampleApp/Players/GeoblockingSkinViewController.m]: recommended  instead of reccomended

**Comment by brol1dev** [comment]
we needed it too in the candidate branch

**Pull request merged**